core/templates/pages/creator-dashboard-page/suggestion-modal-for-creator-view/suggestion-modal-for-creator-view.directive.html

```diff
@@ -24,11 +24,11 @@ <h3>Review Suggestion</h3>
       </div>
     </div>
   </section>
-  <div style="margin-top: 20px; margin-left: 10px;" ng-show="canReviewActiveThread">
+  <div class="oppia-review-message-extra" ng-show="canReviewActiveThread">
```

Comment:
Don't know what you mean by "extra". 

oppia-review-message-input-container perhaps.

Reply:
Actually -extra is used throughout, on many styles. It kinda confused me too.